package.json

```
@@ -9,10 +9,10 @@
     "build": "vite build",
     "build:beta": "vite build --mode beta",
     "preview": "vite preview",
-    "test": "vitest run --project pre && vitest run --project main",
-    "test:cov": "vitest run --project pre && vitest run --project main --coverage",
-    "test:watch": "vitest run --project pre && vitest watch --project main --coverage",
-    "test:silent": "vitest run --project pre && vitest run --project main --silent",
+    "test": "vitest run --project main",
+    "test:cov": "vitest run --project main --coverage --no-isolate",
+    "test:watch": "vitest watch --project main --coverage --no-isolate",
+    "test:silent": "vitest run --project main --silent --no-isolate",
```

```suggestion
    "test": "vitest run",
    "test:cov": "vitest run --coverage --no-isolate",
    "test:watch": "vitest watch --coverage --no-isolate",
    "test:silent": "vitest run --silent --no-isolate",
```
With the pre-test removed you don't need `--project` any more.